server/app/views/admin/TranslationFormView.java

```diff
@@ -103,10 +106,29 @@ protected final FormTag renderTranslationForm(
    * Returns a div containing the default text to be translated. This allows for admins to more
    * easily identify which text to translate.
    */
-  protected final DivTag defaultLocaleTextHint(LocalizedStrings localizedStrings) {
+  protected final DivTag fieldWithDefaultLocaleTextHint(
+      DomContent field, LocalizedStrings localizedStrings) {
     return div()
-        .withClasses(Styles.W_2_3, Styles.ML_2, Styles.P_2, Styles.TEXT_SM, Styles.BG_GRAY_200)
-        .with(p("Default text:").withClass(Styles.FONT_MEDIUM), p(localizedStrings.getDefault()));
+        .withClasses(Styles.GRID, Styles.GAP_6, Styles.GRID_COLS_2)
+        .with(
+            field,
+            div()
+                .withClasses(Styles.PX_2, Styles.PY_1, Styles.TEXT_SM, Styles.BG_GRAY_100)
+                .with(
+                    p("English text:").withClass(Styles.FONT_MEDIUM),
+                    p().with(defaultLocaleTextContent(localizedStrings))));
+  }
+
+  private final ImmutableList<DomContent> defaultLocaleTextContent(
+      LocalizedStrings localizedStrings) {
+    // Convert newlines to break statements.
+    ImmutableList<DomContent> content =
```

Comment:
The `pre` element will handle this for us right without modifying their content?

What do we do for the static question type and other block inputs?

Reply:
Good call on using `<pre>`. For static questions, we actually run it through our own pre-processor that creates accordions, etc. I don't think we'll want to do the same here given that we're only sending plain-text email. `<pre>` is a great way to preserve the original content and I switched to that!